integration/src/ethereum/ethereum.ts

```diff
@@ -243,6 +243,52 @@ export function ethereumTests(get: () => { wallet: core.HDWallet; info: core.HDW
       TIMEOUT
     );
 
+    test(
+      "ethSignTx() - AVAX",
+      async () => {
+        if (!wallet) return;
+        // only test keepkey AVAX signing
+        // eslint-disable-next-line jest/no-jasmine-globals, jest/no-disabled-tests
+
+        const txToSign = {
+          addressNList: core.bip32ToAddressNList("m/44'/60'/0'/0/0"),
+          nonce: "0x01",
+          gasPrice: "0x1dcd65000",
+          gasLimit: "0x5622",
+          value: "0x2c68af0bb14000",
```

Comment:
Kind of a weird test transaction with eth value send along with transfer contract data. I prefer more realistic test data, but tests pass with matching mocks as it stands so won't block over it, just making note.

Reply:
👍